phoenix-core/src/it/java/org/apache/phoenix/end2end/ParameterizedIndexUpgradeToolIT.java

```diff
@@ -111,18 +119,13 @@ public void setup () throws Exception {
     }
 
     private void setClusterProperties() {
-        // we need to destroy the cluster if it was initiated using property as true
-        if (Boolean.toString(upgrade).equals(clientProps
-                .get(QueryServices.INDEX_REGION_OBSERVER_ENABLED_ATTRIB))
-                || Boolean.toString(!isNamespaceEnabled).equals(serverProps
-                .get(QueryServices.IS_NAMESPACE_MAPPING_ENABLED))) {
-            tearDownMiniClusterAsync(1);
-        }
+        tearDownMiniClusterAsync(1);
+
         //setting up properties for namespace
         clientProps.put(QueryServices.IS_NAMESPACE_MAPPING_ENABLED,
-                Boolean.toString(isNamespaceEnabled));
+                Boolean.toString(true));
```

Comment:
Namespaces are not enabled by default, right? Should we set these to false instead?

Reply:
Sure but I think it is good to test with namespace just to cover that too. We need to pick either. I thought this is a safer approach to test that scenario too.

Reply:
I am not sure which ones gives better coverage. If we are not sure, we can leave the isNameSpaceEnabled and set it to true/false on the rows from lines 235-238, as the last values in tuples: e.g. something like this.
            {false, false, true},
            {true, false, false},
            {false, true, false},
            {true, true, true}